tests/BuildSystem/Build/stale-file-removal.llbuild

```diff
@@ -0,0 +1,24 @@
+# Check the handling of stale file removal.
+#
+#
+# RUN: rm -rf %t.build
+# RUN: mkdir -p %t.build
+# RUN: cp %s %t.build/build.llbuild
+# RUN: %{llbuild} buildsystem build --serial --chdir %t.build > %t.initial.out
+# RUN: %{FileCheck} --input-file=%t.initial.out %s
```

Comment:
What is this test really covering? It doesn't really seem like it validates the stale file removal does anything?